src/code/utils/js-plumb-diagram-toolkit.coffee

```diff
@@ -226,11 +230,16 @@ module.exports = class DiagramToolkit
     if opts.showIndicators? and not opts.showIndicators
       changeIndicator = null
 
+    if opts.isTransfer
+      @kit.importDefaults
+        Connector: ["Flowchart", {}]
+        Anchor:    ["Continuous", { faces:["left","right"] }]
+
```

Comment:
I wrote this but now I'm wondering if we need an else here to reset the importDefaults to the real defaults?